csrc/alias_analysis.h

```diff
@@ -61,25 +61,23 @@ class AliasAnalysisResult {
 
   std::string toString(int indent_size) const;
 
-  // Gets the nearest aliased fusion input/output of a `fusion_out` other than
-  // `fusion_out` itself. Returns null if that doesn't exist.
-  TensorView* getNearestAliasedIo(const TensorView* fusion_out) const;
+  // Returns the mapped value in `alias_to_root_` or null.
+  TensorView* getNearestAliasedIo(const TensorView* alias) const;
 
  private:
-  // Same as `getNearestAliasedIo` except that the `get` method returns the
-  // cached result.
-  TensorView* findNearestAliasedIo(TensorView* fusion_out) const;
```

Comment:
I inlined this function to `finalize` because whether it respects the layout wasn't immediately clear to the reader.